src/Components/Form.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { ImInfo } from "react-icons/im";
+function Form() {
+  const [breed, setBreed] = useState("");
+  const [weight, setWeight] = useState("");
+  const [height, setHeight] = useState("");
+  const [isInfo, setIsInfo] = useState(false);
```

Comment:
can you explain what this variable is?

Reply:
Breed, weight, height are all variables for the form.
isInfo is used for conditional rendering, whenever the user presses the information icon it basically does a NOT transaction on it, so it turns off and on whenever the user clicks it.